tests/test_admin_functions.py

```diff
@@ -0,0 +1,83 @@
+from admin import *
+
+
+def test_add_drink():
+    drinks = {}
+    name = "Lager"
+    price = 34
+    add_drink(drinks, name, price)
+
+    assert name in drinks
+    assert drinks[name] == price
+
+
+def test_remove_drink():
+    drinks = {}
+    name = "Lager"
+    price = 40
+    add_drink(drinks, name, price)
```

Comment:
You are testing the remove function. So create a ready-made dictionary with meaning. It is not necessary to use other functions in the test